detection/persistence/unexpected-chrome-extensions.sql

```diff
@@ -339,7 +339,9 @@ WHERE
     'true,Yuri Konotopov <ykonotopov@gnome.org>,GNOME Shell integration,gphhapmejobijbbhgpjhcjognlahblep',
     'true,,Zoom,hmbjbjdpkobdjplfobhljndfdfdipjhg',
     'true,,ZoomInfo Engage Chrome Extension,mnbjlpbmllanehlpbgilmbjgocpmcijp',
-    'true,,Zoom Scheduler,kgjfgplpablkjnlkjmjdecgdpfankdle'
+    'true,,Zoom Scheduler,kgjfgplpablkjnlkjmjdecgdpfankdle',
+    'true,AgileBits,1Password \xE2\x80\x93 Password Manager,aeblfdkhhhdcdjpifhhbdiojplfjncoa',
+    'true,,Loom \xE2\x80\x93 Screen Recorder & Screen Capture,liecbddmkiiihnedobmlmillhodjkdmb'
```

Comment:
The hex codes may not work; you may want to test this in the Live Query page on Kolide if not.

Reply:
Good to know!